console

```diff
@@ -16,6 +20,29 @@ $application = require __DIR__ . '/res/bootstrap.php';
 
 $cli = $application->cli();
 
+$cli->command('serve', function (\Stratify\Http\Application $http, ContainerInterface $container, OutputInterface $output) {
+    $loop = React\EventLoop\Factory::create();
+
+    $db = $container->get(Connection::class);
+
+    $server = new Server(function (ServerRequestInterface $request) use ($http, $db) : ResponseInterface {
+        // Reset the DB connection
+        $db->close();
```

Comment:
Very interesting link thank you! I was starting to consider whether to abstract DBAL behind another abstraction that restarts connections automatically… But that's overkill I think, and I don't think DBAL has an interface for the `Connection`…